### ISSUE TITLE
Switchbot battery

### ISSUE THREAD

**Issue by matt8707** [fphammerle/switchbot-mqtt#41]
Hi, is receiving battery info supported?

Currently using https://github.com/OpenWonderLabs/python-host

---

With [pySwitchbot](https://github.com/Danielhiversen/pySwitchbot), but that library likes to randomly throw errors

```py
import switchbot
sb = switchbot.Switchbot(mac="aa:bb:cc:dd:ee:ff")
print(sb.get_device_data()["data"].get("battery"))
```

**Comment by fphammerle** [comment]
Hi, adding support to receive the battery state is currently blocked by the issue described here: https://github.com/fphammerle/switchbot-mqtt/pull/31#issuecomment-840845203

In summary: `get_device_data` calls `BLEScanner().start()`. Unfortunately, scanning for low-energy bluetooth devices requires elevated privilleges (incompatible with typical docker setups)

**Comment by fphammerle** [comment]
new release `v2.0.0` reports battery levels on topic `homeassistant/cover/switchbot/MAC_ADDRESS/battery-percentage`  (enable with command-line option `--fetch-device-info`)

please open a new issue, if that doesn't work for you
